keps/sig-node/127-user-namespaces/README.md

```diff
@@ -486,6 +482,17 @@ inside the container:
 - `spec.initContainers[*].securityContext.runAsUser`
 - `spec.ephemeralContainers[*].securityContext.runAsUser`
 
+Further, validation will also be relaxed for pods in a baseline namespace:
+
+- `spec.containers[*].securityContext.capabilities`
+- `spec.initContainers[*].securityContext.capabilities`
+- `spec.ephemeralContainers[*].securityContext.capabilities`
+
+The validation for capabilities can be relaxed in a baseline pod because capabilities
+are user namespaced in the linux kernel, and any pod does not have a seccomp profile (as baseline
+pods are not required to) has access to the `unshare` syscall, allowing a user to create a user
```

Comment:
> any pod does not have a seccomp profile (as baseline pods are not required to)

how does this interact with https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2413-seccomp-by-default where a pod that does not opt into a seccomp profile gets one applied at the kubelet level?

does this ability to add arbitrary capabilities undo protections a cluster administrator put in place by defaulting seccomp on at the node level?

Reply:
Since PSA doesn't require RuntimeDefault in baseline, even if RuntimeDefault is on (which needs a kubelet flag to enable) a sufficiently knowledgeable user can just set it to Unconfined and still be in baseline.

Reply:
> a sufficiently knowledgeable user can just set it to Unconfined and still be in baseline

baseline disallows explicitly setting to Unconfined

Reply:
ah I see, I misunderstood that section. okay, so if kubelet sets SeccompDefault, then this would technically increase the capabilities of a container. That said, I still think user namespaced capabilities are safer than not user namespaced capabilities. summoning @giuseppe who can probably advocate for that better than me

Reply:
It's odd that we allow `nil` or unset seccompProfile but not `Unconfined`. Is there a difference?

Reply:
Unconfined explicitly opts out of seccomp

nil / unset allows https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2413-seccomp-by-default to take effect

Reply:
Ah fair point. Thanks!